Marlin/Marlin_main.cpp

```diff
@@ -1691,6 +1691,11 @@ static void do_blocking_move_to(float x, float y, float z, float feed_rate = 0.0
   feedrate = old_feedrate;
 }
 
+inline void do_blocking_move_to_axis(AxisEnum axis, float where, float feed_rate = 0.0) {
```

Comment:
> From a semantics standpoint

I kind-of get it. You're moving `to_x` or `to_y` and those are axes, so how about `do_blocking_move_to_axis_pos`?